### PR TITLE
[TRANSLATION] Croatian translation of games (Solitare and Spider)

### PR DESCRIPTION
**Pull request by Andrej123456789**
## Purpose

Translated Solitare and Spider games to Croatian

JIRA issue: N/A

### CODE REVIEW

File: base/applications/games/spider/lang/hr-HR.rc [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    CONTROL "", IDC_CARDBACK5, "Static", SS_NOTIFY, 232, 7, 50, 71
+    CONTROL "", IDC_CARDBACK6, "Static", SS_NOTIFY, 289, 7, 50, 71
+    CONTROL "", IDC_CARDBACK7, "Static", SS_NOTIFY, 4, 89, 50, 71
+    CONTROL "", IDC_CARDBACK8, "Static", SS_NOTIFY, 61, 89, 50, 71
+    CONTROL "", IDC_CARDBACK9, "Static", SS_NOTIFY, 118, 89, 50, 71
+    CONTROL "", IDC_CARDBACK10, "Static", SS_NOTIFY, 175, 89, 50, 71
+    CONTROL "", IDC_CARDBACK11, "Static", SS_NOTIFY, 232, 89, 50, 71
+    CONTROL "", IDC_CARDBACK12, "Static", SS_NOTIFY, 289, 89, 50, 71
+    DEFPUSHBUTTON "U redu", IDOK, 114, 165, 54, 14
+    PUSHBUTTON "Cancel", IDCANCEL, 176, 165, 54, 14
```
Comment (HBelusca) [inline base/applications/games/spider/lang/hr-HR.rc:31]:
```suggestion
    PUSHBUTTON "Odustani", IDCANCEL, 176, 165, 54, 14
```

File: base/applications/games/solitaire/lang/hr-HR.rc [hunk 2]
```diff
@@ -0,0 +1,93 @@
+/*
+ * PROJECT:     Solitaire
+ * LICENSE:     Freeware, permission to use under Public Domain
+ * FILE:        base/applications/games/solitaire/lang/en-US.rc
+ * PURPOSE:     Croatian Language File for Solitaire
+ * TRANSLATOR: Andrej Bartulin
+ */
+
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
```
Comment (HBelusca) [inline base/applications/games/solitaire/lang/hr-HR.rc:9]:
```suggestion
LANGUAGE LANG_CROATIAN, SUBLANG_CROATIAN_CROATIA
```

File: base/applications/games/spider/lang/hr-HR.rc [hunk 3]
```diff
@@ -0,0 +1,81 @@
+/*
+ * PROJECT:    Spider Solitaire
+ * LICENSE:    See COPYING in top level directory
+ * FILE:       base/applications/games/spider/lang/en-US.rc
+ * PURPOSE:    Croatian Language File for Spider Solitaire
+ * TRANSLATOR: Andrej Bartulin
+ */
+
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
```
Comment (HBelusca) [inline base/applications/games/spider/lang/hr-HR.rc:9]:
```suggestion
LANGUAGE LANG_CROATIAN, SUBLANG_CROATIAN_CROATIA
```

File: base/applications/games/solitaire/lang/hr-HR.rc [hunk 4]
```diff
@@ -0,0 +1,93 @@
+/*
+ * PROJECT:     Solitaire
+ * LICENSE:     Freeware, permission to use under Public Domain
+ * FILE:        base/applications/games/solitaire/lang/en-US.rc
+ * PURPOSE:     Croatian Language File for Solitaire
+ * TRANSLATOR: Andrej Bartulin
```
Comment (HBelusca) [inline base/applications/games/solitaire/lang/hr-HR.rc:6]:
```suggestion
 * FILE:        base/applications/games/solitaire/lang/hr-HR.rc
 * PURPOSE:     Croatian Language File for Solitaire
 * TRANSLATOR:  Andrej Bartulin
```

File: base/applications/games/spider/lang/hr-HR.rc [hunk 5]
```diff
@@ -0,0 +1,81 @@
+/*
+ * PROJECT:    Spider Solitaire
+ * LICENSE:    See COPYING in top level directory
+ * FILE:       base/applications/games/spider/lang/en-US.rc
+ * PURPOSE:    Croatian Language File for Spider Solitaire
+ * TRANSLATOR: Andrej Bartulin
```
Comment (HBelusca) [inline base/applications/games/spider/lang/hr-HR.rc:6]:
```suggestion
 * PROJECT:     Spider Solitaire
 * LICENSE:     See COPYING in top level directory
 * FILE:        base/applications/games/spider/lang/hr-HR.rc
 * PURPOSE:     Croatian Language File for Spider Solitaire
 * TRANSLATOR:  Andrej Bartulin
```

File: base/applications/games/spider/lang/hr-HR.rc [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+ */
+
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
+
+/* Dialogs */
+
+IDD_CARDBACK DIALOGEX 6, 6, 345, 185
+CAPTION "Select Card Back"
```
Comment (HBelusca) [inline base/applications/games/spider/lang/hr-HR.rc:14]: Missing translation.

File: base/applications/games/spider/lang/hr-HR.rc [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+    CONTROL "", IDC_CARDBACK10, "Static", SS_NOTIFY, 175, 89, 50, 71
+    CONTROL "", IDC_CARDBACK11, "Static", SS_NOTIFY, 232, 89, 50, 71
+    CONTROL "", IDC_CARDBACK12, "Static", SS_NOTIFY, 289, 89, 50, 71
+    DEFPUSHBUTTON "U redu", IDOK, 114, 165, 54, 14
+    PUSHBUTTON "Cancel", IDCANCEL, 176, 165, 54, 14
+END
+
+IDD_DIFFICULTY DIALOGEX 100, 100, 106, 80
+STYLE DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU | DS_SHELLFONT
+CAPTION "Difficulty"
```
Comment (HBelusca) [inline base/applications/games/spider/lang/hr-HR.rc:36]: Missing translation.

File: base/applications/games/solitaire/lang/hr-HR.rc [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+ */
+
+LANGUAGE LANG_ENGLISH, SUBLANG_ENGLISH_US
+
+/* Dialogs */
+
+IDD_OPTIONS DIALOGEX 0, 0, 186, 118
+STYLE DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU | DS_SHELLFONT
+CAPTION "Options"
```
Comment (HBelusca) [inline base/applications/games/solitaire/lang/hr-HR.rc:15]: Missing translation.

File: base/applications/games/solitaire/lang/hr-HR.rc [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+FONT 8, "MS Shell Dlg"
+BEGIN
+    GROUPBOX "Karice", -1, 7, 7, 90, 40
+    AUTORADIOBUTTON "Nacrtaj &jednu", IDC_OPT_DRAWONE, 14, 19, 70, 10, WS_GROUP | WS_TABSTOP
+    AUTORADIOBUTTON "Nacraj &tri", IDC_OPT_DRAWTHREE, 14, 32, 70, 10
+    GROUPBOX "Rezlutat", -1, 100, 7, 75, 53
```
Comment (HBelusca) [inline base/applications/games/solitaire/lang/hr-HR.rc:21]:
```suggestion
    GROUPBOX "Rezultat", -1, 100, 7, 75, 53
```

File: base/applications/games/solitaire/lang/hr-HR.rc [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+    AUTORADIOBUTTON "&Ništa", IDC_OPT_NOSCORE, 107, 45, 60, 10
+    AUTOCHECKBOX "&Pokaži vrijeme", IDC_OPT_SHOWTIME, 7 ,51 ,65 ,10, WS_TABSTOP
+    AUTOCHECKBOX "Sta&tusna traka", IDC_OPT_STATUSBAR, 7, 66, 64, 10, WS_TABSTOP
+    AUTOCHECKBOX "&Zadrži rezlutat", IDC_OPT_KEEPSCORE, 100, 66, 65, 10, WS_TABSTOP
+    DEFPUSHBUTTON "U redu", IDOK, 35, 97, 50, 14
+    PUSHBUTTON "Odustani", IDCANCEL, 101, 97, 50, 14
+END
+
+IDD_CARDBACK DIALOGEX 6, 6, 345, 185
+CAPTION "Select Card Back"
```
Comment (HBelusca) [inline base/applications/games/solitaire/lang/hr-HR.rc:33]: Missing translation.

File: base/applications/games/solitaire/lang/hr-HR.rc [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+/* Strings */
+
+STRINGTABLE
+BEGIN
+    IDS_SOL_NAME "Solitaire"
+    IDS_SOL_ABOUT "Solitaire od J Brown\n\nCardLib verzija 1.0."
+    IDS_SOL_QUIT "Napusti trenutnu igru?"
+    IDS_SOL_WIN "Čestitke, pobijedili ste!"
+    IDS_SOL_DEAL "Ponovi?"
+    IDS_SOL_SCORE "Rezluat: %d"
```
Comment (HBelusca) [inline base/applications/games/solitaire/lang/hr-HR.rc:62]:
```suggestion
    IDS_SOL_SCORE "Rezultat: %d"
```

File: base/applications/games/spider/lang/hr-HR.rc [hunk 12]
```diff
@@ -0,0 +1,81 @@
...
+FONT 8, "MS Shell Dlg"
+BEGIN
+    AUTORADIOBUTTON "Easy: &Jedna boja", IDC_DIF_ONECOLOR, 8, 10, 80, 10, WS_GROUP | WS_TABSTOP
+    AUTORADIOBUTTON "Medium: &Dvije boje", IDC_DIF_TWOCOLORS, 8, 23, 80, 10
+    AUTORADIOBUTTON "Hard: Č&etiri boje", IDC_DIF_FOURCOLORS, 8, 36, 80, 10
```
Comment (HBelusca) [inline base/applications/games/spider/lang/hr-HR.rc:41]: Easy/Medium/Hard : Not translated?

File: base/applications/games/solitaire/lang/hr-HR.rc [hunk 13]
```diff
@@ -0,0 +1,93 @@
+/*
+ * PROJECT:     Solitaire
+ * LICENSE:     Freeware, permission to use under Public Domain
+ * FILE:        base/applications/games/solitaire/lang/en-US.rc
+ * PURPOSE:     Croatian Language File for Solitaire
+ * TRANSLATOR: Andrej Bartulin
+ */
```
Comment (binarymaster) [inline base/applications/games/solitaire/lang/hr-HR.rc:7]:
```suggestion
/*
 * PROJECT:     ReactOS Solitaire
 * LICENSE:     Freeware, permission to use under Public Domain
 * PURPOSE:     Croatian resource file
 * TRANSLATORS: Copyright 2022 Andrej Bartulin <andrej.bartulinvr@gmail.com>
 */
```

File: base/applications/games/spider/lang/hr-HR.rc [hunk 14]
```diff
@@ -0,0 +1,81 @@
+/*
+ * PROJECT:    Spider Solitaire
+ * LICENSE:    See COPYING in top level directory
+ * FILE:       base/applications/games/spider/lang/en-US.rc
+ * PURPOSE:    Croatian Language File for Spider Solitaire
+ * TRANSLATOR: Andrej Bartulin
+ */
```
Comment (binarymaster) [inline base/applications/games/spider/lang/hr-HR.rc:7]:
```suggestion
/*
 * PROJECT:     Spider Solitaire
 * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
 * PURPOSE:     Croatian resource file
 * TRANSLATORS: Copyright 2022 Andrej Bartulin <andrej.bartulinvr@gmail.com>
 */
```

**Review by binarymaster — COMMENTED** [review]
Correct headers

File: base/applications/games/solitaire/lang/hr-HR.rc [hunk 15]
```diff
@@ -0,0 +1,93 @@
...
+    GROUPBOX "Karice", -1, 7, 7, 90, 40
+    AUTORADIOBUTTON "Nacrtaj &jednu", IDC_OPT_DRAWONE, 14, 19, 70, 10, WS_GROUP | WS_TABSTOP
+    AUTORADIOBUTTON "Nacraj &tri", IDC_OPT_DRAWTHREE, 14, 32, 70, 10
+    GROUPBOX "Rezlutat", -1, 100, 7, 75, 53
+    AUTORADIOBUTTON "&Standarno", IDC_OPT_STANDARD, 107, 19, 60, 10, WS_GROUP | WS_TABSTOP
+    AUTORADIOBUTTON "&Vegas", IDC_OPT_VEGAS, 107, 32, 60, 10
+    AUTORADIOBUTTON "&Ništa", IDC_OPT_NOSCORE, 107, 45, 60, 10
+    AUTOCHECKBOX "&Pokaži vrijeme", IDC_OPT_SHOWTIME, 7 ,51 ,65 ,10, WS_TABSTOP
+    AUTOCHECKBOX "Sta&tusna traka", IDC_OPT_STATUSBAR, 7, 66, 64, 10, WS_TABSTOP
+    AUTOCHECKBOX "&Zadrži rezlutat", IDC_OPT_KEEPSCORE, 100, 66, 65, 10, WS_TABSTOP
```
Comment (HBelusca) [inline base/applications/games/solitaire/lang/hr-HR.rc:27]:
```suggestion
    AUTOCHECKBOX "&Zadrži rezultat", IDC_OPT_KEEPSCORE, 100, 66, 65, 10, WS_TABSTOP
```

**Review by HBelusca — CHANGES_REQUESTED** [review]

File: base/applications/games/spider/lang/hr-HR.rc [hunk 16]
```diff
@@ -0,0 +1,81 @@
...
+END
+
+/* Strings */
+
+STRINGTABLE
+BEGIN
+    IDS_SPI_NAME "Spider"
+    IDS_SPI_ABOUT "Spider Solitaire od Gregor Schneider\n\nCardLib verzija 1.0"
+    IDS_SPI_QUIT "Napusti trenutnu igru?"
+    IDS_SPI_WIN "Česitike, pobijedili ste!"
```
Comment (HBelusca) [inline base/applications/games/spider/lang/hr-HR.rc:53]:
```suggestion
    IDS_SPI_WIN "Čestitke, pobijedili ste!"
```

File: base/applications/games/solitaire/lang/hr-HR.rc [hunk 17]
```diff
@@ -0,0 +1,92 @@
...
+/* Dialogs */
+
+IDD_OPTIONS DIALOGEX 0, 0, 186, 118
+STYLE DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU | DS_SHELLFONT
+CAPTION "Opcije"
+FONT 8, "MS Shell Dlg"
+BEGIN
+    GROUPBOX "Karice", -1, 7, 7, 90, 40
+    AUTORADIOBUTTON "Nacrtaj &jednu", IDC_OPT_DRAWONE, 14, 19, 70, 10, WS_GROUP | WS_TABSTOP
+    AUTORADIOBUTTON "Nacraj &tri", IDC_OPT_DRAWTHREE, 14, 32, 70, 10
```
Comment (HBelusca) [inline base/applications/games/solitaire/lang/hr-HR.rc:19]:
```suggestion
    AUTORADIOBUTTON "Nacrtaj &tri", IDC_OPT_DRAWTHREE, 14, 32, 70, 10
```

File: base/applications/games/solitaire/lang/hr-HR.rc [hunk 18]
```diff
@@ -0,0 +1,92 @@
...
+    GROUPBOX "Rezultat", -1, 100, 7, 75, 53
+    AUTORADIOBUTTON "&Standarno", IDC_OPT_STANDARD, 107, 19, 60, 10, WS_GROUP | WS_TABSTOP
```
Comment (HBelusca) [inline base/applications/games/solitaire/lang/hr-HR.rc:21]:
```suggestion
    AUTORADIOBUTTON "&Standardno", IDC_OPT_STANDARD, 107, 19, 60, 10, WS_GROUP | WS_TABSTOP
```

**Review by HBelusca — COMMENTED** [review]
Should be OK after those two other modifications.

File: base/applications/games/solitaire/lang/hr-HR.rc [hunk 19]
```diff
@@ -0,0 +1,92 @@
...
+LANGUAGE LANG_CROATIAN, SUBLANG_CROATIAN_CROATIA
+
+/* Dialogs */
+
+IDD_OPTIONS DIALOGEX 0, 0, 186, 118
+STYLE DS_MODALFRAME | WS_POPUP | WS_CAPTION | WS_SYSMENU | DS_SHELLFONT
+CAPTION "Opcije"
+FONT 8, "MS Shell Dlg"
+BEGIN
+    GROUPBOX "Karice", -1, 7, 7, 90, 40
```
Comment (HBelusca) [inline base/applications/games/solitaire/lang/hr-HR.rc:17]:
```suggestion
    GROUPBOX "Kartice", -1, 7, 7, 90, 40
```

**Pull request merged**